network/monuments.csv

```diff
@@ -75,6 +75,7 @@ LKTA,50233M001,Forced Centering,Pillar,-1.289,Reinforced Concrete,2,2004-02-01T0
 LOK1,,Unmarked Mark,Unknown,-0.899,Unknown,0,2016-11-15T00:00:00Z,2017-11-02T00:00:00Z,,
 LOOK,,Forced Centering,Wyatt/Agnew Drilled-Braced,-1.592,Stainless Steel Rods,10,2017-03-28T00:00:00Z,9999-01-01T00:00:00Z,Basalt,Lookout Formation - Trachy-Basalt And Andesite Lava Flows With Soil Cover
 LRR1,,Forced Centering,Steel Mast,0,Stainless Steel Rod,0,2016-11-15T00:00:00Z,9999-01-01T00:00:00Z,,
+LY2T,,Forced Centering,Steel Mast,-2.1,Reinforced Concrete,0,2024-08-22T00:00:00Z,9999-01-01T00:00:00Z,Siltstone,
```

Comment:
Have you got this monument height from the RIM team or estimated from photos?

Reply:
I had a chat with the RIM team. They didn't measure at the site but based on the photos and their memories of putting it up, we got this measurement. I would say it's accurate +/- 0.1 metres

Reply:
That is fine for now. Could you please ask the guys to measure the height next time they visit that site (I believe they have a plan to visit it soon)?

Reply:
Sounds good, I'll let them know